src/routes/api/tripRoute.js

```diff
@@ -239,4 +250,72 @@ tripRouter.post('/twoWay',
  *
  */
 
+tripRouter.patch('/:tripRequestId', verifyToken, Validate.approveOrRejectRequest(), checkInputDataError, Exists.isTripRequestExist, isManagerRequest, isManagerAccess, validateAcceptOrReject, acceptOrRejectRequests);
+/**
```

Comment:
@william000000, I would like to know these two validations you used: `Validate.approveOrRejectRequest()` and `validateAcceptOrReject`

Reply:
one is coming from validation folder, other one is a middleware for this route